book-crawler/scripts/backup_db.py

```
@@ -0,0 +1,222 @@
+            
+            # 백업 실행
+            with open(backup_path, 'w') as f:
+                result = subprocess.run(cmd, stdout=f, stderr=subprocess.PIPE, text=True)
+                
+            if result.returncode != 0:
+                logger.error(f"백업 실패: {result.stderr}")
+                return None
+                
+            # 백업 파일 압축
+            compressed_path = backup_path.with_suffix('.sql.gz')
+            subprocess.run(['gzip', str(backup_path)])
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)